scripts/src/models/applications.ts

```diff
@@ -19,6 +20,10 @@ export class Application extends CreationDateModel {
 
 	@Column("simple-json", { name: "jwt_public_keys" })
 	public jwtPublicKeys!: StringMap;
+
+	@ManyToMany(type => Offer)
```

Comment:
Don't you need to specify the owner field on the Offer class?

Reply:
not owner, but applications. It's a many-to-many relation between apps (digital services) and offers.
The owner_id is a one-to-many relation and stored in `OfferOwner`.
You don't have to declare the opposite relation if you don't use it.